src/Action/LoginAction.php

```diff
@@ -67,6 +70,7 @@ public function __construct(
         TemplateRegistryInterface $templateRegistry,
         TokenStorageInterface $tokenStorage,
         FormFactoryInterface $formFactory,
+        TranslatorInterface $translator,
```

Comment:
Please move this argument to the last position and make it nullable to respect BC

Reply:
also the tests seems to say that 10+ parameters are problematic, should i ignore that?

Reply:
> also the tests seems to say that 10+ parameters are problematic, should i ignore that?

You can ignore this kind of error